### DISCUSSION TITLE
Not being able to re-application even being applicable

### DISCUSSION THREAD

**Discussion by BobbyESP** [community/community#116970] [GitHub Education] [answered]
### Hi everyone,

Hello people! I have a question. It turns out that I am already a beneficiary of the program but I cannot apply again since my center is not among those that appear on the list. I was previously in another educational center, but I have changed as I have advanced in my studies. Before I was at the IES of Meaño (where I did the initial application, and even renewed once), but now I am at the IES of Vilalonga, and, as I said before, it is not present on the list. The institute does not provide us with emails with personalized domains, so I have to use my personal one, which makes what I want to achieve even more impossible. My GitHub PRO ends in 6 days and the truth is that I would be interested in continuing to keep it without paying since I have the possibility.

### IES de Meaño appears and my already submitted attempts in the past
![image](https://github.com/community/community/assets/60316747/226eaf1c-db5e-4b28-8f68-9d7ad36a42da)

### The new school doesn't appear
![image](https://github.com/community/community/assets/60316747/ad22ba80-cede-4d39-ac70-ba7c51fb81a9)

Thanks everyone in advance!

**Comment by samus-aran** [comment, marked as answer]
I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education.

**Reply by BobbyESP** [reply]
Thanks! I forgot to update the thread ❤️
